File: klasa 2/python/funkcje.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  szablon.py

#zasięg zmiennych: lokalny, globalny

def dodaj(a, b):
    return(a + b)

def odejmij(a, b):
    return(a - b)
    
def pomnoz(a, b):
    return(a * b)
    
def podziel(a, b):
    return(a / b)
    
            
def main(args):
    #a = 0 #inicjacja zmiennej
    a = int(input('Podaj 1. liczbę: '))
    print(a)
    b = int(input('Podaj 2. liczbę: '))
    print(b)
    
    suma = float(a) + float(b)
    roznica = float(a) - float(b)
    iloraz= float(a) / float(b)
    iloczyn = float(a) * float(b)
    
    
    print('Suma:', dodaj(a, b))
    print('Różnica:', odejmij(a, b))
    print('Iloraz:', podziel(a, b))
    print('Iloczyn:', pomnoz(a, b))
    
    
   
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: klasa 2/c++/funkcje.cpp
#include <iostream>

using namespace std;

int dodaj(int a, int b)
{
    return a + b;   
}

int odejmij(int a, int b)
{
    return a - b;   
}

int pomnoz(int a, int b)
{
    return a * b;   
}

float podziel(float a, float b)
{
    return a / b;   
}




int main(int argc, char **argv)
{
	
    float a, b; //deklaracja zmiennej
    a = b = 0; //inicjacja zmiennej
    
    
    cout << "Podaj 1. liczbę: ";
    cin >> a; //std:: <- standardowa biblioteka
    cout << a << endl;
    
    cout << "Podaj 2. liczbę: ";
    cin >> b; 
    cout << b << endl;
    
    
    
    cout << "Suma: " << dodaj(a, b) << endl;
    cout << "Różnica: " << odejmij(a, b) << endl;
    cout << "Iloczyn: " << pomnoz(a, b) << endl;
    cout << "Iloraz: " << podziel(a, b) << endl;
        
	return 0;
}

